Updating NDX to 1.0.0-rc1-update1

alpaix opened the pull request:
- Merged @emgarten xunit changes from PR #118 
- Changed build.ps1 script to always install new ndx version
- Upgraded xunit runner dependency to match ndx version (from here https://xunit.github.io/docs/getting-started-dnx.html)
- Resolved deadlock in command line unit-tests.

emgarten committed:
Updating to xunit 2.1.0

emgarten committed:
Remove unit test workaround

alpaix committed:
Upgraded SDK to rc1-u1

alpaix committed:
Resolved unit-test deadlock

dnfclas commented:
Hi **@alpaix**, I'm your friendly neighborhood .NET Foundation Pull Request Bot (You can call me DNFBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla2.dotnetfoundation.org.
    </span>

TTYL, DNFBOT;

alpaix commented on src/NuGet.Clients/NuGet.CommandLine/Commands/Command.cs:
```diff
@@ -120,7 +120,12 @@ public void Execute()
+
+                // Will block until the task is completed...
+                task.Wait();
             }
```
This is the magic that sends async task to a threadpool while the main thread is blocked on waiting.

dnfclas commented:
**@alpaix**, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, DNFBOT;

alpaix committed:
Fixed broken NuGet.Clients unit-tests

+ Resolved deadlock in a unit-test by making it async.
+ Upgraded tests projects to xunit 2.1.0.
+ Added cache clean statement where needed.

emgarten commented:
Looks good :shipit:

davidfowl commented:
Is this getting merged soon?

alpaix commented:
Merged into dev.
Thanks for review!
